lib/si-frontend-types-rs/src/audit_log.rs

```diff
@@ -70,6 +75,22 @@ pub enum AuditLogDeserializedMetadata {
     },
 }
 
+impl AuditLogDeserializedMetadata {
+    pub fn display_name_and_entity_type(&self) -> (&'static str, &'static str) {
+        type Discrim = AuditLogDeserializedMetadataDiscriminants;
+
+        let discrim: Discrim = self.into();
+        match discrim {
+            Discrim::CreateComponent => ("Created", "Component"),
+            Discrim::DeleteComponent => ("Deleted", "Component"),
+            Discrim::UpdatePropertyEditorValue => ("Updated property in Component", "Property"),
+            Discrim::UpdatePropertyEditorValueForSecret => {
+                ("Updated secret in Component", "Property for Secret")
+            }
+        }
+    }
+}
```

Comment:
Right now, these aren't stored on the persisted logs themselves. We may want to change this, but this works for fast iteration on the feature before it is released.

Reply:
@stack72 this might be useful if we want to tune the exact wording of things in the UI for a bit. This will require an SDF compile, so we could also move this to the frontend too.